### PR TITLE
optimize: rename zk configuration root path.

### PR DESCRIPTION
**Pull request by objcoding**
### Ⅰ. Describe what this PR did
There are many projects for 'config' node in zk, such as kafka.

### CODE REVIEW

File: config/seata-config-core/src/main/java/io/seata/config/ConfigurationKeys.java [hunk 1]
```diff
@@ -29,7 +29,7 @@
     /**
      * The constant FILE_ROOT_CONFIG.
      */
-    public static final String FILE_ROOT_CONFIG = "config";
+    public static final String FILE_ROOT_CONFIG = "seata";
```
Comment (xingfudeshi) [inline config/seata-config-core/src/main/java/io/seata/config/ConfigurationKeys.java]: There are some places need to be changed too.

Comment (objcoding) [reply config/seata-config-core/src/main/java/io/seata/config/ConfigurationKeys.java]: > There are some places need to be changed too.

Has been modified.

**Comment by objcoding** [comment]
> There are some places need to be changed too.

Has been modified

File: config/seata-config-zk/src/main/java/io/seata/config/zk/ZookeeperConfiguration.java [hunk 2]
```diff
@@ -47,7 +48,7 @@
 
     private static final String CONFIG_TYPE = "zk";
     private static final String ZK_PATH_SPLIT_CHAR = "/";
-    private static final String ROOT_PATH = ZK_PATH_SPLIT_CHAR + FILE_ROOT_CONFIG;
+    private static final String ROOT_PATH = ZK_PATH_SPLIT_CHAR + SEATA_FILE_ROOT_CONFIG;
```
Comment (zjinlei) [inline config/seata-config-zk/src/main/java/io/seata/config/zk/ZookeeperConfiguration.java]: please check other Configuration Centers and assess if modifications are needed,such as etcd3.

Comment (objcoding) [reply config/seata-config-zk/src/main/java/io/seata/config/zk/ZookeeperConfiguration.java]: > please check other Configuration Centers and assess if modifications are needed,such as etcd3.

only zk

**Pull request merged**